test_data/import_order/expected_representation.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Placeholder_1 do
+  alias X.{K, K.T, Y}
+  alias X.{Z}
+  alias X.Y.{A, Z}
+  alias X.Y.{A, Z}
```

Comment:
All the `alias` with multi-aliases come first. This might not be the most natural way of ordering them, but it makes the code a bit simpler. All we require is consistency anyway.